wasmbinding/message_plugin.go

```diff
@@ -1065,7 +1065,7 @@ func (m *CustomMessenger) resubmitFailure(ctx sdk.Context, contractAddr sdk.AccA
 		return nil, nil, nil, errors.Wrap(sdkerrors.ErrNotFound, "no failure found to resubmit")
 	}
 
-	err = m.ContractmanagerKeeper.ResubmitFailure(ctx, contractAddr, failure)
+	err = m.ContractmanagerKeeper.DoResubmitFailure(ctx, contractAddr, failure)
```

Comment:
We decided to call only MsgServer methods from wasmbindings and no keeper calls, why did you change this?

Reply:
Well it was already Keeper call, I just renamed it. I will try to use msg_server method tho